src/main/java/frc/robot/RobotContainer.java

```diff
@@ -225,6 +229,26 @@ private void configureBindings() {
                         })));
   }
 
+  private boolean IsEndGame() { // Endgame starts when match time is 20 seconds or less
```

Comment:
Ideally if your code is structured well, you should not need comments. The code should explain itself just by reading it like a sentence. Try reading through the next ~15 lines again, and I think you'll see that there's a lot of repeated text in the comments.

I would reserve comments for when you need a longer explanation or something exceptional is occurring and needs emphasis.